mrp_unbuild_rebuild_variant/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module adds a new `mrp.unbuild.rebuild.variant` which acts as a wrapper around the unbuilding - rebuilding process of different variants.
```

Comment:
This is quite useful as it ensures the same tracked components will be used in the rebuilding process.